app/build/resources/linux/Mailspring.desktop.in

```diff
@@ -10,3 +10,103 @@ StartupWMClass=<%= productName %>
 Categories=GNOME;GTK;Network;Email;
 Keywords=email;internet;
 MimeType=x-scheme-handler/mailto;x-scheme-handler/mailspring;
+Actions=NewMessage
+
+[Desktop Action NewMessage]
+Name=New Message
+Name[af]=Nuwe boodskap
+Name[am]=አዲስ መልዕክት
+Name[ar]=رسالة جديدة
+Name[ast]=Mensaxe nuevu
+Name[az]=Yeni mesaj
+Name[be]=Новы ліст
+Name[bg]=Ново съобщение
+Name[bn]=নতুন বার্তা
+Name[br]=Kemennadenn nevez
+Name[bs]=Nova poruka
+Name[ca]=Nou missatge
+Name[co]=Nu missaghju nova
+Name[cs]=Nová zpráva
+Name[cy]=Neges Newydd
+Name[da]=Ny besked
+Name[de]=Neue E-Mail
+Name[el]=Νέο μήνυμα
+Name[en]=New Message
+Name[eo]=Nova Mesaĝo
+Name[es]=Mensaje nuevo
+Name[et]=Uus kiri
+Name[eu]=Mezu berria
+Name[fa]=پیام جدید
+Name[fi]=Uusi viesti
+Name[fr]=Nouveau message
+Name[fy]=Nij berjocht
+Name[ga]=Teachtaireacht Nua
+Name[gd]=Teachdaireachd ùr
+Name[gl]=Nova menxase
+Name[gu]=નવો સંદેશો
+Name[ha]=Sabon Saƙon
+Name[he]=הודעה חדשה
+Name[hi]=नया संदेश
+Name[hr]=Nova poruka
+Name[ht]=Nouvo mesaj
+Name[hu]=Új üzenet
+Name[hy]=Նոր նամակ
+Name[id]=Pesan Baru
+Name[ig]=Ozi ọhụrụ
+Name[is]=Nýr póstur
+Name[it]=Nuovo messaggio
+Name[ja]=新規メッセージ
+Name[ka]=ახალი წერილი
+Name[kk]=Жаңа хабарлама
+Name[km]=សារ​ថ្មី
+Name[kn]=ಹೊಸ ಸಂದೇಶ
+Name[ko]=새로운 메세지
+Name[ku]=Peyama Nû
+Name[ky]=Жаңы кат
+Name[lb]=Neien Message
+Name[lo]=ຂໍ້ຄວາມໃຫມ່
+Name[lt]=Naujas laiškas
+Name[lv]=Jauns ziņojums
+Name[mg]=Hafatra vaovao
+Name[mi]=Karere Hou
+Name[mk]=Нова порака
+Name[mn]=Шинэ захиа
+Name[ms]=Mesej Baru
+Name[mt]=Messaġġ ġdid
+Name[my]=စာအသစ်
+Name[nb]=Ny melding
+Name[ne]=नयाँ सन्देश
+Name[nl]=Nieuw bericht
+Name[nr]=Umlayezo Omutjha
+Name[oc]=Nouveau message
+Name[pa]=ਨਵਾਂ ਸੁਨੇਹਾ
+Name[pl]=Nowa wiadomość
+Name[pt_BR]=Nova Mensagem
+Name[pt]=Nova Mensagem
+Name[rm]=Nov messadi
+Name[ro]=Mesaj nou
+Name[ru]=Новое сообщение
+Name[si]=නව ලිපියක්
+Name[sk]=Nová správa
+Name[sl]=Novo sporočilo
+Name[sm]=Faʻamatalaga fou
+Name[so]=Farriin Cusub
+Name[sq]=Mesazh i Ri
+Name[sr]=Nuevo mensaje
+Name[ss]=Umlayeto lomusha
+Name[st]=Molaetsa o Motjha
+Name[sv]=Nytt brev
+Name[ta]=புதிய செய்தி
+Name[th]=ข้อความใหม่
+Name[tr]=Yeni İleti
+Name[ts]=Hungu Rintshwa
+Name[uk]=Новий лист
+Name[uz]=Yangi xabar
+Name[ve]=Mulaedza Muswa
+Name[vi]=Thư mới
+Name[xh]=Umyalezo omtsha
+Name[zh_CN]=新邮件
+Name[zh_TW]=新增郵件
+Name[zh]=新信息
+Name[zu]=Umyalezo omusha
+Exec=/usr/bin/mailspring mailto:
```

Comment:
Wow I looked through this PR twice before I noticed this line, it's cool this didn't require any new code! I didn't realize these shortcuts could just execute a script.